au4/src/projectscene/view/timeline/timelinecontext.cpp

```diff
@@ -13,6 +13,10 @@ static constexpr double ZOOM_MIN = 0.001;
 static constexpr double ZOOM_MAX = 6000000.0;
 static constexpr int PIXELSSTEPSFACTOR = 5;
 
+//! TODO Make better name
+constexpr double INSURE_VISIBLE_GAP_PX(16);
```

Comment:
from the name it looks like this value will be used only in the insureVisible method - I suggest moving these constants to the method and then maybe it will be easier to choose a name